.github/workflows/tests.yml

```
@@ -41,6 +41,9 @@ jobs:
     - name: Creating Zulip API key
       run: echo -e "${{ secrets.ZULIP_API_KEY }}" >> download
 
+    - name: Exporting Secret Key
+      run: echo -e "SECRET_KEY=${{ secrets.GITHUB_TOKEN }}" >> .env
```

```suggestion
      run: echo -e "SECRET_KEY=${{ secrets.SECRET_KEY }}" >> .env
```
Since this can be a random string 🤔 
Github token may or may not be needed in the future because of GitHub logins, not sure 🤔 So it's better to name this on repositories end, in a different way.